src/App/Pages/Accounts/HomePage.xaml.cs

```diff
@@ -10,23 +10,28 @@ namespace Bit.App.Pages
 {
     public partial class HomePage : BaseContentPage
     {
-        private bool _runInit;
+        private bool _checkRememberedEmail;
         private readonly HomeViewModel _vm;
         private readonly AppOptions _appOptions;
         private IBroadcasterService _broadcasterService;
 
-        public HomePage(AppOptions appOptions = null)
+        public HomePage(AppOptions appOptions = null, bool checkRememberedEmail = true)
         {
             _broadcasterService = ServiceContainer.Resolve<IBroadcasterService>("broadcasterService");
             _appOptions = appOptions;
             InitializeComponent();
             _vm = BindingContext as HomeViewModel;
             _vm.Page = this;
-            _runInit = true;
-            _vm.StartLoginAction = () => Device.BeginInvokeOnMainThread(async () => await StartLoginAsync());
+            _vm.CheckHasRememberedEmail = checkRememberedEmail;
+            _vm.StartLoginAction = async () => await StartLoginAsync();
```

Comment:
This can potentially crash the application if an exception is raised on the `StartLoginAsync` given that it won't be bubbled up to the call because it is an `Action` and not a `Func<Task>`. In this case I don't think it will happen but if you want to be safe, now that we're calling it from an `AsyncCommand`, change the `Action StartLoginAction` on the VM to `Func<Task> StartLoginAsync` thus we don't break the async/await call chain